src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -216,7 +216,7 @@ For more details on session replay, see the following sections:
     * Make sure your browser agent meets these requirements:
       * Agent version v1.260.0 or greater. We recommend [upgrading to the latest agent version](/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes) as we continue making improvements.
       * Agent type: Pro or Pro+SPA. Session replay does not work with the Lite browser agent.
-      * [Session tracking](/docs/browser/browser-monitoring/page-load-timing-resources/cookie-collection-session-tracking/#settings) and session tracing are enabled.
+      * [Session tracing](/docs/browser/browser-monitoring/browser-pro-features/session-traces-explore-webpages-life-cycle/#requirements) is enabled.
```

Comment:
❗ I did not attempt to update the links on the translated pages for this one.  Updates still needed on the translated pages.